lib/newman_conway.js

```diff
@@ -1,9 +1,19 @@
 
-// Time Complexity:
-// Space Complexity:
+// Time Complexity: O(n)
+// Space Complexity: O(n)?...not feeling confident about this because .slice() makes a copy
 
 function newmanConway(num) {
```

Comment:
👍 Slice does make a copy and your `  return result.slice(1).join(' ');` creates a new string with all the numbers in it.